web/src/containers/admin/CreateUser.js

```diff
@@ -0,0 +1,223 @@
+          password: '',
+          passwordScore: 0,
+          passwordQuality: 'Weak',
+          state: ''
+        });
+      })
+      .catch(() => {
+        this.setState({ fetching: false });
+      });
+  };
+
+  render() {
+    const {
+      fetching,
+      name,
+      email,
+      password,
+      passwordScore,
+      passwordQuality,
+      showPassword,
+      state
+    } = this.state;
+
+    // The password percent bar thing is 88% width, so scale
+    // the password score thing accordingly.  That's all.
+    const passwordPercent = 88 * passwordScore;
+
+    // Really the whole password meter should be replaced.
+
+    let passwordColor = 'bg-red';
+    if (passwordScore > 0.5) {
+      passwordColor = 'bg-yellow';
+    }
+    if (passwordScore === 1) {
+      passwordColor = 'bg-green';
+    }
+
+    return (
+      <Fragment>
+        <header className="clearfix px2 py1 bg-white">
+          <div className="left">
+            <Link to="/" className="btn px0 bold caps">
+              {t('titleBasic')}
+            </Link>
+          </div>
+        </header>
+
+        <div className="mx-auto my3 p2 sm-col-6 md-col-4 bg-white rounded">
+          <h1 className="mt0 h2">Create account</h1>
+          <form onSubmit={this.handleSubmit}>
+            <div className="mb2">
+              <label htmlFor="name">Name</label>
+              <input
+                id="name"
+                type="text"
+                name="name"
+                className="input"
+                value={name}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb2">
+              <label htmlFor="email">Email</label>
+              <input
+                id="email"
+                type="text"
+                name="email"
+                className="input"
+                value={email}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb2">
+              <div className="right">
+                <label htmlFor="showPassword">Show password</label>
+                <input
+                  type="checkbox"
+                  checked={showPassword}
+                  onChange={this.toggleShowPassword}
+                />
+              </div>
+              <label htmlFor="password">Password</label>
+              <input
+                id="password"
+                type={showPassword ? 'text' : 'password'}
+                name="password"
+                className="input"
+                value={password}
+                onChange={this.handleChange}
+              />
+              <div className="right" style={{ width: '10%' }}>
+                {passwordQuality}
+              </div>
+              <div
+                className={`inline-block ${passwordColor} p1 rounded`}
+                style={{ width: `${passwordPercent}%` }}
+              />
+              <div className="alert-info p2">
```

Comment:
Can we make this tag a `p` without messing too much up?

Reply:
I reckon that's probably okay.  We'll see what happens...